tests/unit/handlers/SuiChainHandler.test.ts

```
@@ -0,0 +1,759 @@
+  describe('initializeL2', () => {
+    it('should initialize Sui client with proper configuration', async () => {
+      await (handler as any).initializeL2();
+
+      expect((handler as any).suiClient).toBeDefined();
+      expect((handler as any).keypair).toBeDefined();
+      expect((handler as any).suiWormholeContext).toBeDefined();
+    });
+
+    it('should warn when L2 RPC is not configured', async () => {
+      const configWithoutL2Rpc = { ...mockSuiConfig };
+      delete (configWithoutL2Rpc as any).l2Rpc;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)